app/actions/home.js

```diff
@@ -191,4 +202,17 @@ export function getRedditList(subReddit){
 			redditSearchSuccess(subReddit, [], true);
 		});
 	}
+}
+// Have to test this, just stubbing the function for now
+export function postPlaylist(userID, name, tracks){
+	return(dispatch, getState)=> {
+		let state = getState();
+		axios({
+		  method:'post',
+		  url:'/api/v1/reddit?subReddit='+subReddit,
+		  body:JSON.stringify({usserId:userID,playlistName:[name],tracks:tracks})
```

Comment:
send the payload like I sent in fav api call. 
{
      playlists: {
             [name]: [tracks]
     }
}

Reply:
I read the comment on top of the method now 💯

Reply:
👍